geoserver/webapp/src/docker/jetty-env.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0"?>
+<!DOCTYPE Configure PUBLIC "-//Jetty//Configure//EN" "http://www.eclipse.org/jetty/configure.dtd">
+<Configure class="org.eclipse.jetty.webapp.WebAppContext">
+   <New id="ds_gwc_geoserver" class="org.eclipse.jetty.plus.jndi.Resource">
+     <Arg>jdbc/gwc</Arg>
+     <Arg>
+       <New class="org.apache.commons.dbcp.BasicDataSource">
+         <Set name="driverClassName">org.postgresql.Driver</Set>
+         <Set name="username">georchestra</Set>
+         <Set name="password">georchestra</Set>
+         <Set name="url">jdbc:postgresql://database/georchestra</Set>
+         <Set name="validationQuery">SELECT 1</Set>
```

Comment:
I don't know if this is relevant here, but we could also use env variables (too bad that we should also have the info in the default.properties from the georchestra datadir)

Reply:
Your right. I'll update to env var + default values.